src/constants.js

```diff
@@ -0,0 +1,230 @@
+const offerTypes = {
```

Comment:
```suggestion
const OfferType = {
```
Енамы называются с большой буквы, и в единственном числе